opengrok-web/src/test/java/org/opengrok/web/messages/MessagesUtilsTest.java

```diff
@@ -18,40 +18,50 @@
  */
 
 /*
- * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2020, Oracle and/or its affiliates. All rights reserved.
  */
 
-package org.opengrok.indexer.web.messages;
+package org.opengrok.web.messages;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.opengrok.indexer.configuration.Project;
 import org.opengrok.indexer.configuration.RuntimeEnvironment;
 
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
 import java.time.Duration;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.SortedSet;
 
 import static org.junit.Assert.*;
 
 public class MessagesUtilsTest {
     RuntimeEnvironment env;
 
-    private MessagesContainer container;
+    private MessagesContainer container = MessagesContainer.getInstance();
 
     @After
-    public void tearDown() {
+    public void tearDown() throws Exception {
+        Constructor<MessagesContainer> constructor = MessagesContainer.class.getDeclaredConstructor();
+        constructor.setAccessible(true);
+        container = constructor.newInstance();
```

Comment:
same as my previous comment, can you introduce public constructor without the RE parameter?